docs/reference/config.rst

```diff
@@ -1,39 +1,37 @@
 Configuration
 =============
 
-Beets has an extensive configuration system that lets you customize nearly
-every aspect of its operation. To configure beets, you create a file called
-``config.yaml``. The location of the file depend on your platform (type ``beet
-config -p`` to see the path on your system):
-
-* On Unix-like OSes, write ``~/.config/beets/config.yaml``.
-* On Windows, use ``%APPDATA%\beets\config.yaml``. This is usually in a
-  directory like ``C:\Users\You\AppData\Roaming``.
-* On OS X, you can use either the Unix location or ``~/Library/Application
+Beets has an extensive configuration system which lets you customize nearly
+every aspect of beets. To configure beets, a configuration file called
+``config.yaml`` must be created. The location of ``config.yaml`` depends on your operating system (OS).
+
+* On a Unix OS, use ``~/.config/beets/config.yaml``.
+* On Windows, use ``%APPDATA%\beets\config.yaml``. ``%APPDATA%`` is usually an environment variable
+  which stores a directory like ``C:\Users\[Your Windows' username]\AppData\Roaming``.
+* On OS X, you can use either the Unix OS's location or ``~/Library/Application
   Support/beets/config.yaml``.
 
-You can launch your text editor to create or update your configuration by
-typing ``beet config -e``. (See the :ref:`config-cmd` command for details.) It
-is also possible to customize the location of the configuration file and even
-use multiple layers of configuration. See `Configuration Location`_, below.
+You can launch your text editor to create or update your ``config.yaml`` by
+executing ``beet config -e``. (See the :ref:`config-cmd` command for details.) The location of ``config.yaml`` 
```

Comment:
Any particular reason why you prefer “executing” to “typing” here? I used “typing” mostly because it sounds friendlier and less clinical. I like the effect it has of reminding me that I’m interacting with beets by typing commands at a prompt.

Reply:
I prefer "executing" over "typing" because just "typing" a command won't actually "execute" the command. "executing" implies that the command is typed _and_ that the command performs its specific function. Are you still set on using "typing"?

Reply:
Sure. But to make the action take place as a human, you have to type it. :smiley: I guess I’m technically relying on some shared cultural context with the user here, where typing something means typing it at a prompt, which is how you get work done.

Reply:
I still think that "execute" is a better choice but I'll switch the phrasing back to "typing"